### PR TITLE
<Tag/> - Add ellipsis to long tags in a container

### PR DESCRIPTION
**Pull request by EgoziE**
fix - https://github.com/wix/wix-style-react/issues/3986

Was verified in Chrome, FireFox, and Safari

### CODE REVIEW

File: src/Tag/Tag.js [hunk 1]
```diff
@@ -161,7 +161,7 @@ Tag.defaultProps = {
   size: 'small',
   removable: true,
   theme: 'standard',
-  wrap: false,
+  wrap: true,
```
Comment (shlomitc) [inline src/Tag/Tag.js]: Isn’t this a breaking change? How will it affect users of it?

File: src/Tag/tests/Tag.visual.js [hunk 2]
```diff
@@ -130,9 +120,13 @@ const tests = [
 tests.forEach(({ describe, its }) => {
   its.forEach(({ it, props, container }) => {
     storiesOf(`Tag/${describe}`, module).add(it, () => (
-      <div {...container}>
-        <Tag {...commonProps} {...props} />
-      </div>
+      <Box direction={'vertical'}>
```
Comment (shlomitc) [inline src/Tag/tests/Tag.visual.js]: Changing the tests meaning you won’t get indication for the previous one that was done. Maybe it worth a separate PR

Comment (EgoziE) [reply src/Tag/tests/Tag.visual.js]: Good idea!
I splitted the PR - https://github.com/wix/wix-style-react/pull/4188

File: src/Tag/Tag.js [hunk 3]
```diff
@@ -21,21 +22,28 @@ const tagToTextSize = {
 class Tag extends WixComponent {
   static displayName = 'Tag';
 
+  componentDidMount() {
+    if (this.props.hasOwnProperty('wrap'))
```
Comment (nitayneeman) [inline src/Tag/Tag.js]: I think that the following is enough:
```suggestion
    if (this.props.wrap)
```

File: src/Tag/Tag.js (hunk 1 above)
Comment (EgoziE) [reply src/Tag/Tag.js]: Finally, we decided to deprecate "wrap".
A deprecation message is displayed when used.

**Pull request merged**